webui/ec2/views.py

```diff
@@ -23,25 +23,24 @@ class UserView (views.APIView):
     permission_classes = (IsAuthenticated,)
 
     def get(self, request, name=None):
-        yourdata = EC2.get_users(name)
+        yourdata = EC2().get_users(name)
         results = UserSerializer(yourdata, many=True).data
         return Response(results)
 
     def delete(self, request, name):
         # Allow only tmp users to be deleted
-        p = re.compile('^{}$'.format(
-            provider_conf.EC2['username_pattern'].replace('{}', '\\d+')))
+        p = re.compile('^{}$'.format(ConfigFile().get(['default', 'username_pattern']).replace('{}', '\\d+')))
         if not p.match(name):
             return HttpResponseForbidden()
 
-        if EC2.delete_user(name):
+        if EC2().delete_user(name):
```

Comment:
so just for my understanding before we were using here some kind of static method of a class and now we creating real class instance to be able to call method ?